resources/lib/sutils.py

```diff
@@ -170,8 +179,8 @@ def add_item_to_library(self, item_path, item_url):
         error = False
         new = False
         if item_path:
-            item_path = xbmc.translatePath( item_path )
-            dir = os.path.dirname(item_path)
+            item_path = os.path.normpath(xbmc.translatePath( item_path ))
```

Comment:
ked to funguje bez toho, tak som to vyhodil a hotovo....
to nacitavanie filmov z XML je fakt super rychle, oplati sa mergnut